include/SFML/System/Time.hpp

```diff
@@ -49,6 +50,13 @@ class Time
     ////////////////////////////////////////////////////////////
     constexpr Time();
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Construct from std::chrono::duration
+    ///
+    ////////////////////////////////////////////////////////////
+    template<class Rep, class Period>
```

Comment:
```suggestion
    template <typename Rep, typename Period>
```
for consistency. Same further below

Reply:
Fixed!